src/main/resources/META-INF/plugin.xml

```diff
@@ -7,34 +7,50 @@
     <depends>com.intellij.modules.platform</depends>
     <depends>JavaScript</depends>
 
-    <resource-bundle>messages.MyBundle</resource-bundle>
+    <resource-bundle>messages.OxcBundle</resource-bundle>
 
     <extensions defaultExtensionNs="com.intellij">
+        <notificationGroup id="Oxc" displayType="BALLOON"/>
         <iconProvider implementation="com.github.oxc.project.oxcintellijplugin.OxcIconProvider"/>
         <platform.lsp.serverSupportProvider
                 implementation="com.github.oxc.project.oxcintellijplugin.lsp.OxcLspServerSupportProvider"/>
 
         <projectConfigurable
                 parentId="tools"
-                instance="com.github.oxc.project.oxcintellijplugin.settings.OxcSettingsConfigurable"
-                id="Settings.Oxc"
+                instance="com.github.oxc.project.oxcintellijplugin.settings.OxcConfigurable"
+                id="com.github.oxc.project.oxcintellijplugin.settings.OxcSettingsConfigurable"
                 nonDefaultProject="false"
-                bundle="messages.MyBundle"
+                bundle="messages.OxcBundle"
                 key="oxc.name"
         />
+        <actionOnSaveInfoProvider id="OxcOnSaveInfoProvider"
+                                  implementation="com.github.oxc.project.oxcintellijplugin.settings.OxcOnSaveInfoProvider"
+                                  order="after FormatOnSaveInfoProvider, after EsLintOnSaveInfoProvider, before FileWatcherOnSaveInfoProvider, before UploadOnSaveInfoProvider"/>
```

Comment:
The general recommendation for Oxlint CLI usage is to execute Oxlint first and then ESLint. Is there any reason the on save action should be different?

Reply:
actionOnSaveInfoProvider refers to the order in the table on the ‘On Save Action’ tab
<img width="1049" alt="Screenshot 2025-04-02 at 1 13 24 PM" src="https://github.com/user-attachments/assets/17688d12-ca19-442e-b66c-ee7feb5e70d9" />